source/includes/_userprovisioning.md

```diff
@@ -2,7 +2,7 @@
 
 Please note that the base URL for user provisioning is
 
-`https://services.1brd.com`
+`https://profile.services.1brd.com`
```

Comment:
shouldnt it be customer.services.1brd.com? profile was the old name of the service

Reply:
It is profile still as customers already use this API and we need to tell them about the change